MapboxNavigation/ExitView.swift

```diff
@@ -12,6 +12,26 @@ class ExitView: UIView {
     static let leftExitImage = UIImage(named: "exit-left", in: .mapboxNavigation, compatibleWith: nil)!.withRenderingMode(.alwaysTemplate)
     static let rightExitImage = UIImage(named: "exit-right", in: .mapboxNavigation, compatibleWith: nil)!.withRenderingMode(.alwaysTemplate)
     
+    
+    @objc dynamic var foregroundColor: UIColor? {
+        didSet {
+            layer.borderColor = foregroundColor?.cgColor
+            imageView.tintColor = foregroundColor
+            exitNumberLabel.textColor = foregroundColor
+            setNeedsDisplay()
+        }
+    }
+    @objc dynamic var borderWidth: CGFloat {
+        didSet {
+            layer.borderWidth = borderWidth
+        }
+    }
+    @objc dynamic var cornerRadius: CGFloat {
```

Comment:
Can we consolidate `cornerRadius` and `borderWidth` in a `StylableView` similar to `StylableButton`?
`foregroundColor` fits perfectly and can stay in `ExitView`.